mlflow/pyfunc/__init__.py

```diff
@@ -89,9 +89,11 @@
 import sys
 from copy import deepcopy
 
+import mlflow
 from mlflow.tracking.fluent import active_run, log_artifacts
 from mlflow import tracking
 from mlflow.models import Model
+from mlflow.pyfunc.model import _save_model, PythonModel, PythonModelContext
```

Comment:
Is there a reason `_save_model` is in `model.py` if the only caller is in this file. If you want to hide it -- move `save_model` to another file and then alias it here

Reply:
Good call, the reason was to produce documentation for `save_model` under the `mlflow.pyfunc` module name, rather than `mlflow.pyfunc.model`. I've instead renamed `mlflow.pyfunc.model._save_model` to `mlflow.pyfunc.model.save_model` and moved `log_model` to the `mlflow.pyfunc.model` module. I've updated the sphinx configuration to document these functions under the `mlflow.pyfunc` namespace, since they are imported.